app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,41 @@
+class FoodsController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    @foods = Food.where(user_id: current_user.id)
```

Comment:
- [ ] It would be nice if we fix the `n+1` query problem. In our case, we could use `@foods = Food.includes(current_user.id)`, this is just an example